requirements.txt

```diff
@@ -1,8 +1,9 @@
+aiohttp==3.4.4
```

Comment:
This looks like a lot of changes, but I took the liberty to sort the dependencies and also adding `flake8` for local linting.

Reply:
Thank you @ThimDeveloper,
I am finishing to review your PR. It will be merged soon.

Reply:
Hey @itsdeka . 
Any update on when this will be merged so I can rebase and update the Changelog for my other PR accordingly? :D